qa/pull-tester/rpc-tests.py

```diff
@@ -146,6 +148,13 @@ def main():
     config = configparser.ConfigParser()
     config.read_file(open(os.path.dirname(__file__) + "/tests_config.ini"))
 
+    # Set up logging handler
+    sh = logging.StreamHandler()
+    sh.setLevel((logging.INFO if args.quiet else logging.DEBUG))
```

Comment:
Maybe only call sh.setLevel if args.quiet is true, because logging.DEBUG can still throw away logs at levels < 10.